Makefile

```diff
@@ -0,0 +1,74 @@
+SHELL := /usr/bin/env bash
+
+# build the registry image
+IMAGE_REGISTRY?=quay.io
+IMAGE_REPOSITORY?=openshift-sre
+IMAGE_NAME?=osd-operators
+CATALOG_NAMESPACE?=openshift-operator-lifecycle-manager
+DOCKERFILE?=./Dockerfile
+CHANNEL?=production
```

Comment:
How are we going to deal with staging or integration testing images? We could consider using the output of `git rev-parse --abbrev-ref HEAD` for channel if branch is !=master, and maintain some a branch called staging with an alpha catalog for example.